docs/index.rst

```diff
@@ -1,20 +1,25 @@
-async-service
-==============================
+.. include:: introduction.rst
 
-Lifecycle management for async applications
-
-Contents
---------
 
 .. toctree::
-    :maxdepth: 3
+   :hidden:
+   :maxdepth: 1
+   :caption: General
+
+   introduction
+   release_notes
 
-    async_service
-    releases
+.. toctree::
+   :hidden:
+   :maxdepth: 1
+   :caption: Fundamentals
 
+   guides
+   api
 
-Indices and tables
-------------------
+.. toctree::
+   :hidden:
```

Comment:
These are all `hidden` to not have the doctree sit in the footer duplicated on the frontpage. Something we should imho also do in the other projects. Having the toc repeated on the front page seems pointless when it is already in the sidebar.